my-app/src/redux/store.js

```diff
@@ -1,5 +1,17 @@
-import { createStore } from 'redux';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { reducer as formReducer } from 'redux-form';
+import thunkMiddleware from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
 
-import reducer from './game/reducers';
+import gameReducer from './game/reducers';
+import loginReducer from './login/reducers';
 
-export const store = createStore(reducer);
+const reducers = {
+  gameReducer,
+  form: formReducer,
+  loginReducer
```

Comment:
Remember that `gameReducer` is a redundant name, we simply call them `game`, or `login`